pkgs/development/python-modules/file_read_backwards/default.nix

```
@@ -0,0 +1,20 @@
+{ lib, buildPythonPackage, fetchPypi, mock }:
+
+buildPythonPackage rec {
+  pname = "file_read_backwards";
```

```suggestion
  pname = "file-read-backwards";
```

Naming should follow PEP 0503 -> https://nixos.org/manual/nixpkgs/unstable/#contributing